qemu.go

```diff
@@ -483,7 +488,42 @@ func (q *qemu) setMemoryResources(podConfig PodConfig) ciaoQemu.Memory {
 		MaxMem: memMax,
 	}
 
-	return memory
+	return memory, nil
+}
+
+func getHostMemorySize() (uint64, error) {
+	f, err := os.Open("/proc/meminfo")
```

Comment:
This function really need a test so I'd either pass this filename as an argument, or use a global that the tests can modify. For an almost identical example (feel free to copy):

- https://github.com/clearcontainers/runtime/blob/master/cc-check.go#L85..L99
- https://github.com/clearcontainers/runtime/blob/master/cc-check_test.go#L45..L90